contracts/test/mocks/GatewayUpgradeMock.sol

```diff
@@ -32,6 +32,10 @@ contract GatewayUpgradeMock is IGateway, IInitializable {
         return address(0);
     }
 
+    function assetFees() external pure returns (AssetFees memory) {
```

Comment:
Lets rename to `tokenTransferFees`, and secondly, not return `AssetFees` type which is an internal implementation detail which I don't want to expose.

I would just return a tuple (registerFee, sendFee).

Reply:
https://github.com/Snowfork/snowbridge/pull/991/commits/f98004c058bf8841b1cc4d99583078e6358ae89c